setup.py

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env python
+import json
 import pathlib
 
 import setuptools
 
+try:
+    import requests
+except ImportError:
+    import sys
+    import subprocess
+
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'requests'])
+    import requests
```

Comment:
this adds `requests` to the setup dependencies, so maybe we could replace the `requests.get` call with a builtin if necessary?